app/models/user.rb

```diff
@@ -3,7 +3,8 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable, :confirmable,
          :recoverable, :rememberable, :validatable, :trackable
+  has_many :companies, dependent: :destroy
   validates :email, presence: true,
                     format: { with: /\A[^@\s]+@[^@\s]+\z/, message: 'must be a valid address' }
-  validates :first_name, :last_name, :password_confirmation, presence: true
+  validates :first_name, :last_name, presence: true
```

Comment:
let's try organise this file a little bit with comments and spacing so not all code is squished together. We would usually group all the validations together, relations, etc. And just above them add a empty line and add a comment saying the heading of the code below.